### ISSUE TITLE
Chectl cannot start Che with openshift OAuth on CRC

### ISSUE THREAD

**Issue by tomgeorge** [eclipse-che/che#15108]
## Summary

```
λ ~/git/chectl/ master* chectl server:start --os-oauth -a operator -p crc -m 
  ✔ Verify Kubernetes API...OK (it's OpenShift)
  ✔ 👀  Looking for an already existing Che instance
    ✔ Verify if Che is deployed into namespace "che"...it is not
 ›   Error: You requested to enable OpenShift OAuth but the platform doesn't seem to be OpenShift. Platform is crc.
```

When installing che via the operator and chectl on crc, the program will not enable openshift oauth because the platform is CRC.  Is there a way for Che to integrate with CRC's OAuth?

**Comment by sleshchenko** [comment]
@tomgeorge as a workaround you can set openshift as platform, OAuth should work, at least it worked for me. But note that you have to set up OAuth on crc (see https://docs.openshift.com/container-platform/4.1/authentication/understanding-identity-provider.html#supported-identity-providers) and loging with anyuser via new oauth (not kubeadmin)
otherwise operator will do nothing but just prints into logs that OAuth must be configured.

And chectl definitely should be fixed.

**Comment by ibuziuk** [comment]
Closing. Fixed on master, 7.3.x and crwctl
